test_integration/lib/test/realtime/realtime_auth_url_test.dart

```diff
@@ -0,0 +1,42 @@
+
+import 'dart:io';
+
+import 'package:ably_flutter/ably_flutter.dart';
+import 'package:ably_flutter_integration_test/app_provisioning.dart';
+import 'package:ably_flutter_integration_test/factory/reporter.dart';
+
+//Make sure that creating realtime with authURL won't crash
+Future<Map<String, dynamic>> testCreateRealtimeWithAuthUrl({
+  required Reporter reporter,
+  Map<String, dynamic>? payload,
+}) async {
+  reporter.reportLog('init start');
+  final appKey = await AppProvisioning().provisionApp();
+
+  final clientOptionsForToken = ClientOptions(
+    key: appKey,
+    environment: 'sandbox',
+    logLevel: LogLevel.verbose,
+  );
+
+  final ablyForToken = Rest(
+    options: clientOptionsForToken,
+  );
+  final accessToken = await ablyForToken.auth.requestToken();
+
+  final Map<String, String> headers = {
+    HttpHeaders.authorizationHeader: "Bearer $accessToken",
+    HttpHeaders.contentTypeHeader: "application/json",
+    HttpHeaders.acceptLanguageHeader: "ios",
+  };
+  final options = ClientOptions(
+    authUrl: 'https://auth.ably.io/auth',
```

Comment:
I don't think `auth.ably.io` is a valid domain. did you mean to mock the http response? it may be easier to just use. the echo server

Reply:
As above

Reply:
Test for connections were added